Updated UCS panel

Dzierzan opened the pull request:
- Updated UCS panel artwork
- Updated offsets for buttons, etc.
- Added extra buttons in the artwork like highlighted variants.

michaeldgg2 commented:
Button variants are not implemented in `chrome.yaml`, please add.

Also right arrow image is "jumping", when a tab is opened, see GIF.

![opene2140_sidebar_tab_bug](https://user-images.githubusercontent.com/119738087/215340032-757959b0-74ed-4428-a299-1cece9cd49be.gif )

Dzierzan commented:
> Also right arrow image is "jumping", when a tab is opened, see GIF.

It was actually your mistake :D. I fixed it.

> Button variants are not implemented in chrome.yaml, please add.

Done! I added also some comments and rearranged that. I hope you don't mind.

michaeldgg2 commented:
You're right, unfortunately I didn't notice it. The sidebar works great now!
